generate-prison-api-types.sh

```diff
@@ -1 +1 @@
-npx openapi-typescript https://api-dev.prison.service.justice.gov.uk/api/swagger.json > server/@types/prisonApiImport/index.d.ts
```

Comment:
Yes I think so. I don't know the history, but the old url no longer works so I updated to the current url and reimported the types. It looks like the last time it was updated was July last year, so this will have picked up any changes to prison api since then